### DISCUSSION TITLE
[Question] Would it be desired that the app creates the teams at the organization level?

### DISCUSSION THREAD

**Discussion by mdelapenya** [repository-settings/app#639] [Q&A] [answered]
Hi 👋 , first thanks for this app, I think it's wonderful having a managed app that controls the settings for multiple repos. Thank you very much for it.

I have one question regarding my own expectations when reading the docs and creating my first settings.yml file:

I've verified that, before the settings.yml file is applied to a repository with the specific grants for each team, I need to manually create them one by one using GH UI, and wonder if it would be possible for the app to declare the teams to be created, representing the single source of truth for the GH organization as code.

**Comment by travi** [comment, marked as answer]
the scope of this app is for configuring repository-level details. organization-level details are out of scope, other than the extension feature for details that are repeated for each repository from an account-level config.

while it could make sense for another app to be made available under this org for the type of features you mention, our team does not have plans to create such an app at this point or the bandwidth to take on such an effort.

**Reply by mdelapenya** [reply]
Thanks for your kind response, I appreciate your work here and will look for another app that manages that config

**Reply by laughedelic** [reply]
@mdelapenya please, share it here if you find it. I'm also interested in it.

**Reply by mdelapenya** [reply]
The one that I would like to use is https://github.com/github/safe-settings, but I think it's not production-ready because there is not enough demand https://github.com/github/safe-settings/issues/260

But I think I'd use it for our organization if existed a hosted version of it
